cmd/eksctl-anywhere/cmd/renewcertificates.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/certificates"
+)
+
+type renewCertificatesOptions struct {
+	configFile string
+	component  string
+}
+
+var rc = &renewCertificatesOptions{}
+
+var renewCertificatesCmd = &cobra.Command{
+	Use:          "certificates",
+	Short:        "Renew certificates",
+	Long:         "Renew external ETCD and control plane certificates",
+	PreRunE:      bindFlagsToViper,
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return rc.renewCertificates(cmd)
+	},
+}
+
+func init() {
+	renewCmd.AddCommand(renewCertificatesCmd)
+	renewCertificatesCmd.Flags().StringVarP(&rc.configFile, "config", "f", "", "Config file containing node and SSH information")
+	renewCertificatesCmd.Flags().StringVarP(&rc.component, "component", "c", "", "Component to renew certificates for (etcd or control-plane). If not specified, renews both.")
+}
```

Comment:
```suggestion
      if err := getPackageCommand.MarkFlagRequired("config"); err != nil {
		log.Fatalf("marking config as required: %s", err)
	}
}
```
so that we don't need to manually check if config is empty